qt/python/mantidqt/mantidqt/widgets/sliceviewer/presenters/lineplots.py

```diff
@@ -127,9 +127,7 @@ def update_line_plot_limits(self):
         """
         # ensure plot labels are in sync with main axes
         self._axx.relim()
-        self._axx.autoscale(axis='y')
```

Comment:
How were these causing a problem? It seems to behave fine on a nightly version for me.

Reply:
It seems the issue is a bit phantom. I went back over the older version and it also seems to work in most cases. There are a few instances where I can trick the extents to be messed up but I cannot currently reliably reproduce it. So for this PR, I will remove these changes until I can better characterize the problem.